modules/tensor_mechanics/include/materials/FiniteStrainHyperElasticViscoPlastic.h

```diff
@@ -7,30 +7,66 @@
 #ifndef FINITESTRAINHYPERELASTICVISCOPLASTIC_H
 #define FINITESTRAINHYPERELASTICVISCOPLASTIC_H
 
+/**
+ * This class solves the viscoplastic flow rate equations in the total form
+ * Involves 5 different types of user objects that calculates:
+ * Internal variable rates - functions of internal variables and flow rates
+ * Internal variables - functions of internal variables
+ * Strengths - functions of internal variables
+ * Flow rates - functions of strengths and PK2 stress
+ * Flow directions - functions of strengths and PK2 stress
+ * PK2 stress - function of Rigth Cauchy Green tensor
+ * The associated derivatives from user objects are assembled and the system is solved using NR
+ **/
+
 #include "ComputeStressBase.h"
-#include "HyperElasticStress.h"
-#include "FlowRateModel.h"
+#include "HEVPStressUOBase.h"
+#include "HEVPFlowRateUOBase.h"
+#include "HEVPStrengthUOBase.h"
+#include "HEVPInternalVarUOBase.h"
+#include "HEVPInternalVarRateUOBase.h"
 
 class FiniteStrainHyperElasticViscoPlastic;
 
 template<>
 InputParameters validParams<FiniteStrainHyperElasticViscoPlastic>();
 
-/**
- * This class is for Hyperelastic Visco-Plastic material
- * Multiple flow models are solved simultaneously
- * The flow models are passed as user objects
- * Residual equations are set for flow rate equations
- * Solved using NR with substepping
- * Stress is calculated using user objects (Present PK2 stress)
- */
-
 class FiniteStrainHyperElasticViscoPlastic : public ComputeStressBase
 {
  public:
   FiniteStrainHyperElasticViscoPlastic (const InputParameters & parameters);
```

Comment:
Sorry for the mundane comment, but it seems that you indented by only 1 space rather than 2 spaces ;-)